Setting/Factions/The Coven.md

```diff
@@ -5,7 +5,7 @@ allies:
 enemies:
   - "[[Ship Defense Corps]]"
   - Intership Council
-demonym: coven
+demonym: granny
```

Comment:
I was looking through the issues and started wondering if Granny Nettles might either belong to this faction, or at least reside in the boroughs as a more independent agent. With this mention I was curious if you were thinking similar?